src/Components/TestExplorer.fs

```diff
@@ -94,10 +94,21 @@ module CancellationToken =
 
         tokenSource.token
 
+
+
 type TestId = string
 type ProjectPath = string
 type TargetFramework = string
 
+module Project =
+    let testPathSubDir = ".ionide-test"
```

Comment:
idea: consider using [`ExtensionContext.storageUri`](https://code.visualstudio.com/api/references/vscode-api#ExtensionContext) from the `activate` call to provide our workspace-unique state path?